app/components/admin/stats/sdg_component.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= back_link_to %>
+
+<%= header %>
+
+<%= render Shared::SDG::GoalStatsComponent.with_collection(goals) %>
```

Comment:
Why `Shared::` instead of `Admin::` :thinking:? How about `Admin::Stats::GoalComponent` or `Admin::Stats::SDG::GoalComponent`?

Reply:
Initially, I thought that probably we would want to render SDG stats at public pages too, but of course, currently, this is potentially a YAGNI thing.